gateway-ha/src/main/java/io/trino/gateway/ha/util/Constants.java

```diff
@@ -1,20 +1,7 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.trino.gateway.ha.handler;
+package io.trino.gateway.ha.util;
+
+public class Constants {
```

Comment:
Put `{` on a separate line and you could add `final`.

Reply:
The name is too general. Please rename to `TrinoHttpUtil` or something.

Reply:
I like the part where it has Trino because this class is meant for Trino http spec.